qemu/tests/virtio_fs_share_data.py

```diff
@@ -166,6 +166,7 @@ def check_socket_group():
             nfs_local.setup()
 
     try:
+        vm = None
```

Comment:
I think you still need a little bit of tweaks about the stage of `finally:`
```
    finally:
        if setup_local_nfs:
            if vm.is_alive():
                vm.destroy()
```
since if it failed to execute `env_process.preprocess`, the last finally stage will still hit the errors about `None object have no atribute is_alive`

Reply:
Thanks, correct again.
if vm and vm.is_alive():